dist/assets/css/main.css

```diff
@@ -937,6 +938,30 @@ i.icon.burger span {
     font-size: 9vw;
   }
 }
+#articles > div > a {
+  display: -webkit-flex;
```

Comment:
Because we use Autoprefixer you don't have to prefix CSS rules. You can simply write `display: flex` and Autoprefixer will add the `display: -webkit-flex; display: -ms-flexbox;` for you.

So story short: we never have to write CSS prefixes. Please remove.